src/views/VallisFish.vue

```diff
@@ -66,33 +63,29 @@
             invert="true"
           />
         </template>
+        <template slot="time" slot-scope="data">
+          <i class="fas fa-times-circle"></i>
+        </template>
         <template slot="rarity" slot-scope="data">
           <FishImg
-            v-if="data.item.rarity === 'Common'"
+            v-if="data.item.rarity === '1'"
             type="common"
             item="common"
             name="Common"
             title="Common"
             width="20"
           />
           <FishImg
-            v-else-if="data.item.rarity === 'Uncommon'"
+            v-else-if="data.item.rarity === '2'"
```

Comment:
hmmmm, i think it would've been better to keep these more like the custom thumbs...

use the "rarity" to determine the other fields that way we don't have a bunch of if/else's

Reply:
I only changed these to numbers because the sorting will depend on the underlying data, not the image itself. If I make the String prefixed with a number, that would work. Because currently if you sort, it is sorting alphabetically.

`Common` -> `Legendary` -> `LOL` -> `Rare` -> `Uncommon`

If I prefix it by a number, it would achieve the same effect.

`0-Peculiar` -> `1-Common` -> `2-Uncommon` -> `3-Rare` -> `4-Legendary`

Reply:
and then trim the beginning 2 characters for the displays?

Reply:
The display for the tooltip is going to be passed in via a separate field on FishImg, so yes that's achievable.